include/errno.h

```diff
@@ -59,6 +59,7 @@
 #define EWOULDBLOCK  EAGAIN  /* Operation would block */
 
 #define ENOTSOCK        88
+#define EMSGSIZE        90
```

Comment:
please note that this errno is defined in libphoenix, so additional PR to remove it from there is needed